compiler/rustc_hir_analysis/src/check/check.rs

```diff
@@ -787,16 +787,6 @@ pub(crate) fn check_item_type(tcx: TyCtxt<'_>, def_id: LocalDefId) {
             for item in items {
                 let def_id = item.id.owner_id.def_id;
 
-                if tcx.has_attr(def_id, sym::rustc_intrinsic) {
-                    intrinsic::check_intrinsic_type(
-                        tcx,
-                        item.id.owner_id.def_id,
-                        item.span,
-                        item.ident.name,
-                        abi,
-                    );
-                }
```

Comment:
So this was for `extern` blocks with functions marked as `#[rustc_intrinsic]`? That was never even a thing, was it?

So yeah I agree with removing this.^^

Reply:
I think it was just something I overlooked in the transition PR where an ABI check was turned into an attr check. Not explicitly added for the attr